demaceo/fitlit
# Iteration1/organizing class files

#### What does  this PR do?

This pull request implements webpack and begins a refactor of several files, most notably userRepository and userRepository-Test.js. The PR debugged tests 4 and 5 in userRepository. HydrationRepository.js was created with the idea that all of the User class' data relating to hydration will be instatiated through the HydrationRepository. This will also make the user class less cluttered. While an initial file has been created, the methods within the HydrationRepository have not been refactored; they have simply been copied from User.js

In similar fashion, throughout User.js, this Pull Request has commented out methods and properties that might be better served within files such as SleepRepository and ActivityRepository as methods within those classes. 


#### Where should the reviewer start?

The reviewer should start by looking at HydrationRepository and comparing the methods within that class to the properties and methods relating to water intake within User.js. The reviewer should then run npm test and verify that the tests within the UserRepository are all clearing. 

#### How should this be manually tested?

The reviewer should run NPM START and then type localHost8080 in a browser to verify that the latest version of the project is up and running on the server. 

#### Any background context you want to provide?

We should consider whether to destroy Sleep.js, Activity.js and Hydration.js and instead convert them to HydrationRepository.js, SleepRepository.js, and ActivityRepository.js instead. 

#### What are the relevant tickets?

[2]-Create webpack server 
[3]-Create an index.scss file
[4]-Create a variable scss file
